app/HutchManager/Controllers/ActivitySourceController.cs

```diff
@@ -35,7 +35,9 @@ public async Task<IActionResult> Delete(int id)
     {
       await _activitySources.Delete(id);
     }
-    catch (KeyNotFoundException) { }
+    catch (KeyNotFoundException) {
+      Console.WriteLine("Could not delete Data Source");
```

Comment:
don't use `Console.WriteLine` in "real" apps. e.g. in an ASP.NET Core app we have dependency injection and a logging framework.

if you actually wanted some output here you would use a dependency injected `ILogger`.

but I'm not sure there's any value in recording this? If an item cannot be found to delete, we usually behave as if everything's ok, since it's functionally already deleted ;)